app/web/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es6",
```

Comment:
We in the future now

Reply:
I wonder if we can be even bolder and have the target at something like `es2020`? Since `es6` is still pretty old (equivalent to `es2015` I believe?) and since the death of IE, pretty much most consumers are on newish version of Chromium via Edge too